packages/@o3r/rules-engine/schematics/ng-add/index.ts

```diff
@@ -81,7 +85,30 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
         ngAddToRun: depsInfo.o3rPeerDeps
       }),
       ...(options.enableMetadataExtract ? [updateCmsAdapter(options)] : []),
-      await registerDevtools(options)
+      await registerDevtools(options),
+      () => {
+        const moduleFilePath = getAppModuleFilePath(tree, context, options.projectName);
+        if (!moduleFilePath) {
+          return tree;
```

Comment:
should we do the equivalent for standalone app ?

Reply:
It already works (the helper name is not really good I agree)